README.md

```diff
@@ -1,99 +1,78 @@
 # The TinyKV Course
 
-This is a series of projects on a key-value storage system built with the Raft consensus algorithm. These projects are inspired by the famous [MIT 6.824](http://nil.csail.mit.edu/6.824/2018/index.html) course but aim to be closer to industry implementations. The whole course is pruned from [TiKV](https://github.com/tikv/tikv) and re-written in Go. After completing this course, you will have the knowledge to implement a horizontally scalable, highly available, key-value storage service with distributed transaction support and a better understanding of TiKV implementation.
+The TinyKV course builds a key-value storage system with the Raft consensus algorithm. It is inspired by [MIT 6.824](https://pdos.csail.mit.edu/6.824/) and [TiKV Project](https://github.com/tikv/tikv).
 
-The whole project is a skeleton code for a kv server and a scheduler server at the beginning, and you need to finish the core logic step by step:
+After completing this course, you will have the knowledge to implement a horizontally scalable, highly available, key-value storage service with distributed transaction support. Also, you will have a better understanding of TiKV architecture and implementation.
 
-- Project1: build a standalone key-value server
-- Project2: build a highly available key-value server with Raft
-- Project3: support multi Raft group and balance scheduling on top of Project2
-- Project4: support distributed transaction on top of Project3
+## Course Architecture
 
-**Important note: This course is still under development, and the documentation is incomplete.** Any feedback and contribution is greatly appreciated. Please see help wanted issues if you want to join in the development.
+The whole project is a skeleton code for a key-value server and a scheduler server at the beginning - you need to finish the core logic step by step:
 
-## Course
+* [Standalone KV](doc/project1-StandaloneKV.md)
```

Comment:
@Whorra I read your summary of TinyKV course and think it is wonderful. Would you mind to help reviewing this course architecture? Maybe elaborate a bit on its content.

Reply:
Okay. 😊